department-of-veterans-affairs/va.gov-cms
# VACMS-4588 4773 small config

## Description
See #4588 #4773

## Testing done
Visual / behat / phpunit

## QA steps
- As an administrator, go to `admin/people/roles` and visually verify that you see `Content creator - Outreach Hub`
- As an administrator, go to `admin/config/search/path/patterns` and visually verify that you do not see Detail page pattern.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
